src/integrationTest/java/uk/gov/hmcts/reform/ccd/documentam/controller/CaseDocumentAmControllerIT.java

```diff
@@ -751,6 +873,22 @@ private Document buildDocument(final Date time, final String caseTypeId) {
             .build();
     }
 
+    private Document buildDocument(final Date time) {
```

Comment:
can we name explicitly like  buildDocumentWithOutCaseId(..)

Reply:
Fixed in b84a992